packages/e2e-tests/specs/experiments/post-editor-template-mode.test.js

```diff
@@ -103,12 +103,15 @@ describe( 'Post Editor Template mode', () => {
 			wp.data.dispatch( 'core/block-editor' ).clearSelectedBlock();
 		} );
 
+		await createNewTemplate( 'Blank Template' );
```

Comment:
I don't think we should add this here because it changes what the test is about, this test is about testing block-based themes where you can switch to the template mode for default templates as well.

Reply:
@youknowriad 

Thanks for reviewing !

If we do not create a new template, the only template that exists is the "Defalut Template". In the case of "Defalut Template", the "Edit" link will not appear, and a timeout will occur. Therefore, we need to create an editable template and link it to the post.

Reply:
For block-based themes, the default template is editable, and that test switches to "tt1-blocks" in the beginning of the test.

Reply:
Sorry, I must have misunderstood. I deleted the new template and it passed the test.